### PR TITLE
Utilize PathJoinSubstitution

### LINKED ISSUE #222: Utilize PathJoinSubstitution
Use PathJoinSubstitution instead of os.path.join

### PR DESCRIPTION
**Pull request by Isopod00**
All of our launch files now use PathJoinSubstitution instead of os.path.join! I tested all of our launch files after implementing this change to make sure that nothing was broken.

### CODE REVIEW

File: src/gstreamer/launch/laptop_launch.py [hunk 1]
```diff
@@ -30,6 +27,7 @@ def generate_launch_description():
         cmd=["rqt", "--force-discover", "--standalone", "CameraClient"], shell=True, output="screen"
     )
 
+    # NOTE: RVIZ uses a lot of bandwidth, so it should be turned off during competition matches
```
Comment (jmblixt3) [inline src/gstreamer/launch/laptop_launch.py:30]: Why was this added in this pr

**Review by jmblixt3 — APPROVED** [review]

File: src/gstreamer/launch/laptop_launch.py (hunk 1 above)
Comment (Isopod00) [reply src/gstreamer/launch/laptop_launch.py:30]: Simply because I forgot to add it earlier / when I was looking at the laptop_launch file I realized I should probably add some sort of note about rviz bandwidth in case we forget or something.
